src/App.tsx

```diff
@@ -1,61 +1,125 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useEffect, useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState([...todosFromServer]);
+  const [task, setTask] = useState('');
+  const [user, setUser] = useState('Choose a user');
+  const [newTitle, setNewTitle] = useState(true);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [selectedUser, setSelectedUser] = useState(true);
```

Comment:
pay attention to variable names
will be better to name it `isUserSelected`, if it has boolean value